internal/appsec/emitter/sqlsec/types/sql.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package types
+
+import (
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo"
+)
+
+type (
+	SQLOperation struct {
+		dyngo.Operation
+	}
+
+	SQLOperationArgs struct {
+		Query  string
+		Driver string
```

Comment:
Can you add the usual comment we have in the other *sec package which states to what sec address they correspond please?

Reply:
3a50d6baa4fe